src/routes/StackView/StackView.tsx

```diff
@@ -3,24 +3,45 @@ import {useDispatch} from "react-redux";
 import {useParams, useNavigate} from "react-router";
 import {useTranslation} from "react-i18next";
 import _ from "underscore";
+import {animated, Transition} from "react-spring";
 import {Color, getColorClassName} from "constants/colors";
 import {NoteDialogComponents} from "components/NoteDialogComponents";
 import {Portal} from "components/Portal";
 import {useAppSelector} from "store";
 import {Actions} from "store/action";
 import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
 import "./StackView.scss";
+import {StackNavigation} from "components/StackNavigation";
+import {useEffect, useRef, useState} from "react";
+import {Note} from "types/note";
+import {AvataaarProps} from "components/Avatar";
+
+type StackedNote = Note & {
+  authorName: string;
+  avatar?: AvataaarProps;
+};
+
+const getTransform = (state: "start" | "end", dir?: string) => {
+  if (dir === "left") {
+    return state === "start" ? "translateX(-100%)" : "translateX(100%)";
+  }
+  if (dir === "right") {
+    return state === "start" ? "translateX(100%)" : "translateX(-100%)";
+  }
+  return "translateX(0)";
+};
 
 export const StackView = () => {
-  const {t} = useTranslation();
   const {boardId, noteId} = useParams();
   const navigate = useNavigate();
   const dispatch = useDispatch();
+  const {t} = useTranslation();
 
   const note = useAppSelector((state) => state.notes.find((n) => n.id === noteId));
-  const column = useAppSelector((state) => state.columns.find((c) => c.id === note?.position.column));
-  const author = useAppSelector((state) => state.participants?.others.find((participant) => participant.user.id === note?.author) ?? state.participants?.self);
-  const authorName = useAppSelector((state) => (author?.user.id === state.participants?.self.user.id ? t("Note.me") : author!.user.name));
+  const prevNote = useRef<Note | undefined>(note);
+  const columns = useAppSelector((state) => state.columns, _.isEqual);
+  const author = useAppSelector((state) => state.participants?.others.find((participant) => participant.user.id === note?.author) ?? state.participants?.self, _.isEqual);
+  const authorName = useAppSelector((state) => (author?.user.id === state.participants?.self.user.id ? t("Note.me") : author!.user.name), _.isEqual);
```

Comment:
Are the `_.isEqual` required here? Thought they'd be for deep comparison, but the values are just strings.

Reply:
I thought passing a function to check equality allows the useAppSelector to memoize the result. Is that not correct? If not I can just remove it

Reply:
doesn't really seem to make any difference. I removed it for now